src/cache.rs

```diff
@@ -217,7 +217,7 @@ impl Cache {
 
         // Get cache dir
         let cache_dir = match Self::get_cache_dir() {
-            Ok((cache_dir, _)) => cache_dir.join("tldr-master"),
+            Ok((cache_dir, _)) => cache_dir.join("tldr-main"),
```

Comment:
This will result in a "cache directory missing" warning, and two caches side by side.

@niklasmohrin what do you think, should we simply keep the name "tldr-master" so that the cache is re-used? Should we rename the old directory if present?

Reply:
I think we should remove the name "tldr-master" too, otherwise the upstream effort would be a pointless. The easiest change would probably be to just remove all the directories with previous names too in `Cache::clear`.

The cleanest approach would be to give the directory a custom name like "pages-cache" or so. This way, there would be less coupling between the downloading and lookup. We could keep the migration code for a year or so and remove it again later

In any case, the migration code should be another PR (tealdeer still works with just this PR)

Reply:
@niklasmohrin is it okay if I leave the PR as is then? I'm having a hiccup with my rust setup lol

Reply:
If CI passes, I am fine with merging :)

Reply:
@niklasmohrin Good point. The migration logic could work like this:

- If the cache is missing (in the `None` branch of `check_cache` in `main.rs`), before showing the warning, first check whether the old cache directory exists, but not the new one. If that is the case, rename the old directory to the new name and log the migration on stderr.
- When explicitly updating the cache, check whether the old directory exists. If yes, simply remove it. This will also handle the cases where both the old and the new cache exist.

Then proceed as before. This way, the check will only happen if the cache is missing, so we don't have a performance penalty.

This can be done in a new PR though.

Reply:
I ran `cargo fmt`, clippy had nothing to say, so things should be fine.